src/control/cmd/dmg/pretty/network.go

```diff
@@ -40,27 +46,56 @@ func PrintHostFabricMap(hfm control.HostFabricMap, out io.Writer, onlyProviders
 	}
 
 	ew := txtfmt.NewErrWriter(out)
+
 	for _, key := range hfm.Keys() {
+		var hfiMap map[uint32]map[string][]string
```

Comment:
nit: No need to predeclare this here... You can just assign to the variable with the result of `make()`.

Reply:
Removed as a result of the other changes to simplify the map handling.